src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -515,25 +292,6 @@ internal TestAccessor(InProcLanguageServer server)
             {
                 _server = server;
             }
-
-            internal ImmutableArray<Uri> GetFileUrisInPublishDiagnostics()
-                => _server._publishedFileToDiagnostics.Keys.ToImmutableArray();
-
-            internal ImmutableArray<DocumentId> GetDocumentIdsInPublishedUris()
-                => _server._documentsToPublishedUris.Keys.ToImmutableArray();
-
-            internal IImmutableSet<Uri> GetFileUrisForDocument(DocumentId documentId)
-                => _server._documentsToPublishedUris.GetValueOrDefault(documentId, ImmutableSortedSet<Uri>.Empty);
-
-            internal ImmutableArray<LSP.Diagnostic> GetDiagnosticsForUriAndDocument(DocumentId documentId, Uri uri)
-            {
-                if (_server._publishedFileToDiagnostics.TryGetValue(uri, out var dict) && dict.TryGetValue(documentId, out var diagnostics))
-                {
-                    return diagnostics;
-                }
-
-                return ImmutableArray<LSP.Diagnostic>.Empty;
-            }
```

Comment:
yaay deleted code.

Reply:
Super happy to delete this code - but this means that in order for razor to get diagnostics, they will have to set the feature flag to LSP diagnostics for regular c# files as well right?  I'm OK with that but wanted to make sure that is the goal